cmd/buildtools/utils.go

```
@@ -114,6 +117,89 @@ func LatestChartVersion(repo, name string) (string, error) {
 	return hcli.Latest(repo, name)
 }
 
+func GetImageDigest(image string) (string, error) {
+	ref, err := docker.ParseReference("//" + image)
+	if err != nil {
+		panic(err)
```

Can we return an error here ?